### PR TITLE
EE: Subsumption Algorithm -- Part 5

### PR DESCRIPTION
**Pull request by aubakirova**
Tests for wildcard lists (`*`)
@mikewest

### CODE REVIEW

**Comment by Sun77789** [comment]
Rebased!

File: content-security-policy/embedded-enforcement/subsumption_algorithm-source_list-wildcards.html [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="support/testharness-helper.sub.js"></script>
+</head>
+<body>
+  <script>
+    var tests = [
+      { "name" : "Wildcard list subsumes an empty source list.",
+        "required_csp": "img-src *",
+        "returned_csp_1": "img-src ",
+        "returned_csp_2": null,
```
Comment (mikewest) [inline content-security-policy/embedded-enforcement/subsumption_algorithm-source_list-wildcards.html]: Nit: Just leave this off.

**Review by mikewest — APPROVED** [review]

**Comment by foolip** [comment]
Oops, this was approved but never merged, probably because of the failing "code-review/reviewable" check. @aubakirova, although there are no conflicts, can you rebase this on master and see if the test still works? That will also trigger another check run, and if it passes we can merge this.

**Comment by foolip** [comment]
@andypaicu, might you be able to revive this PR?

**Comment by gsnedders** [comment]
landed in #27727

**Pull request closed without merging**